src/stratis_cli/_actions/_data.py

```diff
@@ -207,29 +209,54 @@
 DBUS_TIMEOUT_SECONDS = 120
 
 
+def get_timeout():
+    """Get timeout environment variable and return it as an integer"""
```

Comment:
Please format the method doc string as is done elsewhere. We use sphinx markup to generate docs, see; this is why you see a lot of ```:param``` and ```:return``` strings in the docs. Also, ```:raises```.